azure-toolkit-libs/pom.xml

```diff
@@ -48,6 +48,7 @@
         <module>azure-toolkit-database-lib</module>
         <module>azure-toolkit-redis-lib</module>
         <module>azure-toolkit-compute-lib</module>
+        <module>azure-toolkit-containerregistry-lib</module>
```

Comment:
remember to add `azure-toolkit-containerregistry-lib` into `dependencyManagement`